dashboard/test/models/user_test.rb

```diff
@@ -1036,6 +1036,44 @@ def complete_script_for_user(user, script, completed_date = Time.now)
     assert @student.can_edit_email?
   end
 
+  test 'picture_or_word_account? is false for user in no sections' do
+    refute @student.picture_or_word_account?
+  end
+
+  test 'picture_or_word_account? is false for users in picture or word sections with passwords' do
+    picture_section = create(:section, login_type: Section::LOGIN_TYPE_PICTURE)
+    word_section = create(:section, login_type: Section::LOGIN_TYPE_WORD)
+
+    # picture section
+    student_with_password = create(:student, encrypted_password: '123456')
+    create(:follower, student_user: student_with_password, section: picture_section)
+    student_with_password.reload
+    refute student_with_password.picture_or_word_account?
+
+    # word section
+    student_with_password = create(:student, encrypted_password: '123456')
+    create(:follower, student_user: student_with_password, section: word_section)
+    student_with_password.reload
+    refute student_with_password.picture_or_word_account?
```

Comment:
DRY (also below).
```
[Section::LOGIN_TYPE_PICTURE, Section::LOGIN_TYPE_WORD].each do |login_type|
  section = create :section, login_type: login_type
  follower = create :follower, section: section
  refute follower.student_user.picture_or_word_account?
end
```
The default follower uses the default student, which uses the default user (specifying the user_type), which has a password (so presumably encrypted password).